### PR TITLE
Katello changes for pulp 3.11

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
requires  https://github.com/theforeman/foreman-packaging/pull/6498

File: packages/katello/katello-repos/katello-repos.spec [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 %define repo_dir %{_sysconfdir}/yum.repos.d
 %define repo_dist %{dist}
 
 %global prereleasesource nightly
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 2
-
+%global release 3
+6
```
Comment (evgeni) [inline packages/katello/katello-repos/katello-repos.spec:9]:
```suggestion

```

File: packages/katello/katello-repos/katello-repos.spec [hunk 2]
```diff
@@ -68,6 +67,9 @@ rm -rf %{buildroot}
 %config %{repo_dir}/*.repo
 
 %changelog
+* Thu Apr 08 2021 Justin Sherrill <jsherril@redhat.com>
```
Comment (evgeni) [inline packages/katello/katello-repos/katello-repos.spec:70]:
```suggestion
* Thu Apr 08 2021 Justin Sherrill <jsherril@redhat.com> - 4.1-0.3.nightly
```

File: packages/katello/katello-repos/katello-repos.spec [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
+%global pulpcore_version 3.11
 
 %define repo_dir %{_sysconfdir}/yum.repos.d
 %define repo_dist %{dist}
 
 %global prereleasesource nightly
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 2
-
+%global release 3
```
Comment (evgeni) [inline packages/katello/katello-repos/katello-repos.spec:8]:
```suggestion
%global release 3

```

**Comment by evgeni** [comment]
```
DEBUG util.py:600:  No matching package to install: 'rubygem(pulp_deb_client) < 2.11.0'
```

but in #6498 you updated deb_client to 2.11.0 :(

**Comment by jlsherrill** [comment]
i don't know how i screwed that up!

**Comment by jlsherrill** [comment]
@evgeni objections to me untagging 2.11?  I think thats the quickest, simplest solution for now and we can upgrade to 2.11 for server + client later on

**Comment by jlsherrill** [comment]
or actually downgrading it to 2.10 i should say

**Comment by jlsherrill** [comment]
opened: https://github.com/theforeman/foreman-packaging/pull/6527

**Comment by evgeni** [comment]
[test rpm]

**Review by evgeni — APPROVED** [review]

**Pull request merged**